packages/augur-ui/src/modules/trading/store/betslip-hooks.ts

```diff
@@ -248,7 +265,10 @@ export function BetslipReducer(state, action) {
     }
     case MODIFY_BET: {
       const { marketId, orderId, order } = action;
-      const toWin = getNewToWin(getOddsObject(order.normalizedPrice)[ODDS_TYPE.AMERICAN], order.wager);
+      const toWin = getNewToWin(
+        getOddsObject(order.normalizedPrice)[ODDS_TYPE.AMERICAN].value,
```

Comment:
We shouldn't assume American Odds and we also shouldn't ever be doing calculations in odds, that's only for the output and should be handled within the component, not saved to state. if they change odds, that won't update the betslip state and our display should always match their odds. It looks like we had this in place before but now that we know we need to adjust by odds for display we should update this. might get sticky if we start using odds in the background instead of sticking to what we do in trading